runtime/testy/runtimeUtil.py

```diff
@@ -133,3 +135,7 @@ class TimeoutError(Exception):
 
 class RuntimeError(Exception):
   pass
+
+# Sensor type names are CamelCase, with the first letter capitalized as well
+config_file = open(os.path.join(os.path.dirname(__file__), '../../hibike/hibikeDevices.json'), 'r')
```

Comment:
The alternative is to 
1) have hibike auto subscribe?
or
2) hard code what to subscribe to in runtime